### ISSUE TITLE
Properties window uses wrong theme

### ISSUE THREAD

**Issue by vector4x** [files-community/Files#6066]
### Description

When searching to see the properties of the hard disk, the data is made unreadable as there is contrast in the background and the text

### Steps To Reproduce

Open files app
Left click on the hard drive (any)
Select hard drive properties
the properties window opens

### Expected behavior

That the properties of the hard disk can be displayed

### Files Version

Versión: 2.0.1.0 OS Version: 10.0.22000.176

### Windows Version

Windows 11 Pro Versión	21H2  22000.176

### Relevant Assets

![Captura de pantalla (254)](https://user-images.githubusercontent.com/86859564/132182545-7159fd09-074c-451c-9070-727bec4ae484.png)

**Comment by gave92** [comment]
What theme do you have set in Windows? Is it dark theme in windows but light theme in Files?

**Comment by vector4x** [comment]
Dark Mode

**Comment by alexandrehassan** [comment]
I have been trying to reproduce this issue and I haven't been able to, I've tried different combinations of light/dark mode for windows and files, I've also tried different themes (both included and added manually). I've also tried restarting and not restarting Files after the changes.

This might have been solved in between 2.0.1.0 (the version in the issue and my version 2.1.5.0).

Version: 2.1.5.0
OS Version: 10.0.19042.1466
